examples/bri-3/package.json

```diff
@@ -41,7 +41,7 @@
     "@nestjs/platform-express": "^9.0.0",
     "@nestjs/schedule": "^3.0.1",
     "@personaelabs/spartan-ecdsa": "^2.1.4",
-    "@prisma/client": "^4.4.0",
+    "@prisma/client": "^5.2.0",
```

Comment:
Something is strange. Why is this still shown as a change after rebase main? Main already has prisma/client 5.2.0  @Kasshern

Reply:
Yeah something was strange. Perhaps because I branched from another branch that wasn't rebased? unsure. Anyways looks better now. I think...